splitio_android/android/src/main/java/io/split/splitio/SplitWrapperImpl.java

```diff
@@ -222,4 +223,21 @@ public List<SplitView> splits() {
     public SplitView split(String splitName) {
         return mSplitFactory.manager().split(splitName);
     }
+
+    @Override
+    public String getUserConsent() {
+        UserConsent userConsent = mSplitFactory.getUserConsent();
+        if (userConsent == UserConsent.GRANTED) {
+            return "granted";
```

Comment:
why are not you using the defined constant?

Reply:
`UserConsent` is an enum actually. I guess I could return a String representation of the value but I wanted to keep it separate, since I'm checking for these exact Strings in the channel (see [here](https://github.com/splitio/flutter-sdk-plugin/pull/76/files#diff-11798de7d310d7255610e26563c06525b9764e47ddf066e83235aeb47f1d907dR331))